input/standardized-verification/notebook.ipynb

```diff
@@ -0,0 +1,1518 @@
9syaLOqUIGcHwvgiisHayO1o6wniC8c4F7KyP49VMhp1SpAur121PLx2FEVzOxU+R7ufgYhCt3c4/3WtdiyKwKa5h6KhPAFgZ6ZrSLMI8uFw5VoE2ujeeJ6ZRWCMC+TECEL5j+ULJButikIj/RE/ukKcEYtAzhqaDUhFMBnMgsXDURSSmT7qFAudCfrC2XmJ1B9I3tFtCkWB9Us8dHntDMUUTgSsed0prUtVuXgSDvjsY+qotbK6vHa6vHbKbBY85M55r9PlVKU0lk4851oE6I9Y6Alb6PLaWL2gEruS215GPAzisKnyXQOjWBGpZ6IHjPN18LbyLIsgj8LQsLsLxAiMFoF74hZB9rF8FoFRrtBIf6xy04WcNTQrkIpgMiRiJq6hGA7ipm7mhc5kzqyhTq+dpbWuvFM7jbQu8XAsYGNvv50kSt6OsG1JDd0hK3suORhJKKMqGYAaBrEoan06B+ysXVydOz8fqCaE025V5bx2VsyrwDFKJ56NVu8/DKhKR8vSWgirImhpqKJzQL2u1aLkV0DZiiCvRVBuHiPIRyGXz6gW+WRxEQAAIABJREFUgdYhKyadfUj9PkMOQxxAUeuZNLSzbjkohdcHaHmWzALYxUAqglmBVASTwXT6aCwnPqCxyEQRjLUjBFURJITC906pKRHyzQRqTWU9feZtVW60QCyAXUmwakEl+/rtHPLb8tbJogjWNlbTOY5OPBsPQ5RZBC+cKycQs4zr/g/67ewbsLN6QaW+fiKHlLvOQUJ10SWi5jGC7GBxNJQ/RgApi8DE5WOxZ74HZhaB5sopr84NFkfD4PRkykE6DuD0ACIzfqB18E5P4RXDep4lz8y6hmSwuKSRimAy5HENZccHNBY6E/RHrERT/deFYQt9w9axd4SpDv7VC2VcVRGnTDHfeP3qhmqsiuDVC2XUOpJjntrZttTD65cdRJOKfi1TuSUe9vvs+KKWgnL5sCiCtTUxXr2g7uM81jJal3gYSSj8/rKj8DkGi6Bac2/lixHEU20TG06tB5iARZBdtt2lBknj0Uw5q0NVBGYWgXteWs54Dpgf0xSGe94oFoGhjBm1CKQiKGWKrgi6u7u5/fbbaW5upqWlhW3bthW7ClOHyToCXyhWQBGoGkBbS5C9b8Fo1FeUscStugfaas2VAIDTYaW5Wj3eVhcb89TOtiXp7JxtS2vyyxnqazxnPGj1d9uSrJxfOaZzrjHUqa2Q8rSqCsauxPEoWtqIfMHilEWQvR7ADLtL7djihimrYV9u2drfI4aFfZrCcLhNAsJhcNWhuo1Mppm66zP/NtbXVV+4gzeWEQ2rs4iKiXQNzQqKrghsNhv/8A//wNGjR/n973/PP/3TP3HkyJFiV2NqSERyLILAcCxnMZmGpgjOh60kkoLOARsOi2BNQ25CtXy01sYy/p+sXMY5qQ5+XnmChur8mVJbU52/0ypYtaBizOVnlFGn1mtdTRyrZWyaqrHGSV2Z2oYFlWdKOduJpwPeZi4fW3naIjAL+majuZeyR+05FkEeOYdLlTWLAzjcqmIaq0Uw7FPrX1aRGz8wYlQYCNXdVEzkDmWzgqIrgkWLFnHNNdcAUFlZSXNzM729vcWuxtRguqAsmpNnSKPBpf5Y73mthuVf2Mn/OeGmuTpOmW3sP5K2VMfeNkoH31YXH5OckRXzKqi0JWmrjaMUMCMWVpezyJlgXU0Mm3Vir9BY78OIoii01caotCdZVl9AAaWeSRkxapTU+ok8FkE4NMh//12YZMhkGmiOfOpYTgefpTy0MrJH8Ha3emzYYClEBtU4gN2dshbGqgi8KeXhJid+YGTYqyqYsor038UkEVVjKGM1SyUzgiKEKDzBfBo5e/Yst912G4cOHaKqKnNUvH37drZv3w5AT08PO3bsmPT1hoaGqKiY2AjWjOv2fpKwq5HDyz4OQCwJH/9dOe9bMMztC0ZY3NBI7/keXV4I2Ot1EIpbqKqsIjgYZFVljBtWLMqQM2IsY3FDI6e7ezgadLDeE834bWXLvdPTwwG/g9aaKBYTOWe5i+GRTJfC4oZG/u3EBartSdY3NRSs0+sn+yizCK5dXljOWKdsuUMBO03uOKuWLi4oZyyv6/R5BuMW3rNqYV65tWUXWHfwcb5S9bf4B87z9/bt/Pv6rxEpq1ddMKmNZ1b2PIfr/Ou0jXybbyz5d+66/C3evPpvCbmWZshpnz2BI7Qe/zqdrf+TgKcFgJte/3MGqtdyounPdbmK0FnaDz/Gwau/wED9DQBct/cThMoWEnI10tT7U1677f8hLDbKhy9y4xsdHGv6CxouvUbcUcWB9X8HQEPvi6w6uZ29a7/C9Qe/yMkVD9LbeKd6jwcewzHST8/CTTSf/g5vXP/PDLsactqi+cg/UBU4yttL/zNrT/4v3rrm6wxWrTRtN5j638jyt79Lw/mX+W37/05/aWzbbEzafabkhuJWKmyJUeVKse4aj3xpK/v27TM/z8CMRXCGhobYvHkz3/zmN3OUAEBHRwcdHR0AtLe3s2HDhklfc/fu3VNSjs4BO+6Fi9kwXzW3L41YgHL8l97hUH8vWz58H5//4sMZp7hT/7Z+5ht8/osP4++HDQ9+I0dOw1jGlg/fx8tffBgbcPh8YbnPf/FhHMCRPnO5q1uu4dDhP+Qce/mXDzMMfOovCtfp5V8+zCCw4YGx191M7gzQ8dEto8oZrwuwoaPAdT92JxyEzstJrrGoFsFNi5JgC0L7Ztj3NADCsoxQ724A3jg/wl12uG6BAGemnP65DDgObavfBc0bVM3+2xANVTYa5hvOCQs4DGuXL4a2DWoZb0ZxVzqgygq98N7r10PFPDjfCW/Au+usECoHK+l39LW9cBKubyyDgworF9exUjt28nGwltNcZ4HTcMO6VdBosjVdz7cg4mTtfAVOwrVrmmDFBtN2g2n4jYRfhP5y/TcCZLZtNmbtPkNyuy9VZdZ7tPJKqO7jZUZmDcViMTZv3syWLVu46667
```

Comment:
Why make every piece of text in this notebook a second level heading? Also, there are some spelling errors here.